src/Graphics/OpenGLContext/opengl_BufferedDrawer.cpp

```diff
@@ -109,7 +109,7 @@ void BufferedDrawer::_updateRectBuffer(const graphics::Context::DrawRectParamete
 		return;
 	}
 
-	const u32 crc = CRC_Calculate(0xFFFFFFFF, _params.vertices, dataSize);
+	const u64 crc = CRC_Calculate(0xFFFFFFFF, _params.vertices, dataSize);
```

Comment:
`const u64 crc = CRC_Calculate(0xFFFFFFFFFFFFFFFF, _params.vertices, dataSize);`

or better define some constant for that, like
#define CRC_SEED 0xFFFFFFFFFFFFFFFF